src/components/People/People.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable';
+
+export const People: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loader, setLoader] = useState<boolean>();
+  const [errorMassege, setErrorMassege] = useState('');
+
+  useEffect(() => {
+    setLoader(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setErrorMassege('Something went wrong'))
+      .finally(() => setLoader(false));
+  }, []);
+
+  return (
+    <div className="block">
+      <div className="box table-container">
+        {loader && <Loader />}
+
+        {errorMassege && (
```

Comment:
The typo in the state variable `errorMassege` is repeated here. It should be `errorMessage`.